### PR TITLE
cmd/tunnelserver: register to kontrol

### PR DESCRIPTION
**Pull request by rjeczalik**
/cc @ppknap

### CODE REVIEW

File: scripts/check-generated-files.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (cihangir) [inline scripts/check-generated-files.sh]: 👍

**Comment by cihangir** [comment]
```
ensure JSON and generated files in go/src/koding are not stale
cd $WERCKER_SOURCE_DIR
$ export WERCKER_STEP_ROOT="/wercker/steps/wercker/script/0.0.0"
$ export WERCKER_STEP_ID="ba48fcff-f677-427a-8850-af9f5ec5dbc6"
$ export WERCKER_STEP_NAME="script"
$ export WERCKER_REPORT_NUMBERS_FILE="$WERCKER_REPORT_DIR/$WERCKER_STEP_ID/numbers.ini"
$ export WERCKER_REPORT_MESSAGE_FILE="$WERCKER_REPORT_DIR/$WERCKER_STEP_ID/message.txt"
$ export WERCKER_REPORT_ARTIFACTS_DIR="$WERCKER_REPORT_DIR/$WERCKER_STEP_ID/artifacts"
$ mkdir -p $WERCKER_REPORT_ARTIFACTS_DIR
$ export WERCKER_STEP_TEMP="/tmp/$WERCKER_STEP_ID"
$ source '/wercker/wercker-build-essentials/init.sh'
$ mkdir -p $WERCKER_STEP_TEMP
$ scripts/check-generated-files.sh
bootstrap.json.tmpl.go
diff --git a/go/src/koding/klient/tunnel/tlsproxy/pem/bindata.go b/go/src/koding/klient/tunnel/tlsproxy/pem/bindata.go
index 310c2b1..e2b2701 100644
--- a/go/src/koding/klient/tunnel/tlsproxy/pem/bindata.go
+++ b/go/src/koding/klient/tunnel/tlsproxy/pem/bindata.go
@@ -84,7 +84,7 @@ func fullchainPem() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "fullchain.pem", size: 3432, mode: os.FileMode(420), modTime: time.Unix(1474360936, 0)}
+	info := bindataFileInfo{name: "fullchain.pem", size: 3432, mode: os.FileMode(420), modTime: time.Unix(1475512753, 0)}
 	a := &asset{bytes: bytes, info: info}
 	return a, nil
 }
@@ -104,7 +104,7 @@ func privkeyPem() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "privkey.pem", size: 1704, mode: os.FileMode(420), modTime: time.Unix(1474360936, 0)}
+	info := bindataFileInfo{name: "privkey.pem", size: 1704, mode: os.FileMode(420), modTime: time.Unix(1475512753, 0)}
 	a := &asset{bytes: bytes, info: info}
 	return a, nil
 }
````

**Comment by rjeczalik** [comment]
@cihangir Yup, I need to set fixed timestamp for this one.

**Comment by rjeczalik** [comment]
@cihangir @ppknap ping

**Review by cihangir — APPROVED** [review]

**Pull request merged**